src/strict/processors/openai_processor.py

```
@@ -64,3 +67,24 @@ async def process(self, request: ProcessingRequest) -> OutputSchema:
+        ]
+
+        try:
+            stream = await self.client.chat.completions.create(
+                model=self.model,
+                messages=messages,  # type: ignore
+                stream=True,
+            )
+            async for chunk in stream:
+                if chunk.choices[0].delta.content:
+                    yield chunk.choices[0].delta.content
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)